src/app.md

```diff
@@ -189,6 +190,69 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-tar-iterate]: #ex-tar-iterate
+<a name="ex-tar-iterate"></a>
+## Iterate through the entries of a tarball, removing a path prefix
+
+[![flate2-badge]][flate2] [![tar-badge]][tar] [![cat-compression-badge]][cat-compression]
+
+Iterates through the elements of a tarball, removing their path prefix.
+
+Creates an archive, `archive.tar.gz`, using the approach shown in the
```

Comment:
no need to create the archive here (we already have example for it) lets keep the snippets short and to the point.

Just assume existence of `archive.tar.gz`